microsoft-edge/extensions-chromium/getting-started/extension-sideloading.md

```diff
@@ -9,24 +9,44 @@ ms.date: 11/07/2022
 ---
 # Sideload an extension
 
-During development, you can use the Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension.  This article explains how to sideload extensions into Microsoft Edge.
+During development, you can use the Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension.
 
-To sideload an extension:
+To sideload an extension into Microsoft Edge:
 
-1. Open the `edge://extensions` page by choosing the three dots at the top of your browser, and then selecting **Extensions**.
+1. In Microsoft Edge, select **Settings and more** (...) > **Extensions**.
 
    ![Opening the edge://extensions page.](./media/part1-threedots.png)
 
-1. On the extension management page at `edge://extensions`, turn on **Developer mode** using the toggle at the bottom left of the page.
+   The **Extensions** pop-up box opens.
+
+1. Click **Manage extensions**.  The **Extensions** page (`edge://extensions`) opens in a new tab:
 
    ![Turning on Developer Mode.](./media/part1-developermode-toggle.png)
 
-1. When installing your extension for the first time, choose **Load Unpacked**.  You'll be prompted for the directory with your extension source files.  Your extension is installed in your browser, similar to extensions installed from the store.
+1. Turn on **Developer mode** by using the toggle at the bottom left of the page.
+
+1. When installing your extension for the first time, click the **Load Unpacked** button.  You'll be prompted for the directory with your extension source files.  Your extension is installed in your browser, similar to extensions installed from the store.
 
    ![Installed extensions page, showing a sideloaded extension.](./media/part1-installed-extension.png)
 
-During development, you may also need to do the following:
+During development, you may also need to update or remove an extension.
+
+
+<!-- ====================================================================== -->
+## Updating an extension
+
+To update the extension:
+
+1. Go to `edge://extensions`.
+
+1. Select **Reload** on the extension.
+
+
+<!-- ====================================================================== -->
+## Removing an extension
+
+To remove the extension from your browser:
 
-* Update the extension.  Go to `edge://extensions`, and then select **Reload** to update your extension.
+1. Go to `edge://extensions`.
 
-* Remove the extension from your browser.  Go to `edge://extensions`, and then select `Remove` on your extension.
+1. Click **Remove** on the extension.
```

Comment:
nit: Click **Remove** button corresponding to your extension.

Reply:
A card or rectangle is listed for each extension, representing the extension.  I flipped the "click" wording but kept the standard UI wording.  The Remove link/button is on or within the UI area that represents the extension, so "on" is better, simpler, and more standard than "corresponding to", which might be confusingly verbose, if i understand the UI context.